File: Easy/StringToInteger.java
package Easy;

import java.util.Scanner;

public class StringToInteger {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        String str = sc.nextLine();
        sc.close();

        System.out.println(ConvertToInteger(str));
    }

    private static int ConvertToInteger (String str ) {
        try {
            double result = Long.parseLong(str.strip().split(" ")[0]);
            System.out.println(result);
            if(result < Integer.MIN_VALUE) {
                return Integer.MIN_VALUE;
            }
            else if(result > Integer.MAX_VALUE) {
                return Integer.MAX_VALUE;
            }
            else return (int)result;


        } catch (Exception e) {
        }
        return 0;
    }
}

File: Easy/StringToInteger.py
def StringToInteger(s:str) -> int:
    try:
        result = int(s.strip().split()[0])
        if result < -2147483648:
            return -2147483648
        elif result > 2147483647:
            return 2147483647
        else : return result
    except ValueError:
        result = 10

s = input()
print(StringToInteger(s))
        